features/capture-stream-audio-video.yml

```
@@ -0,0 +1,6 @@
+name: captureStream() for <audio> and <video>
+description: The `captureStream()` method for `<audio>` and `<video>` elements returns a MediaStream object which is streaming a real-time capture of the content being rendered in the media element.
```

I'm not really clear on what this does. Does this capture it? (pun intended)

```suggestion
description: The `captureStream()` method for `<audio>` and `<video>` elements returns a `MediaStream` for the media element's content. You can use this to record media or send it elsewhere, such as a canvas or WebRTC connection.
```